.github/workflows/tests.yml

```diff
@@ -42,19 +47,19 @@ jobs:
         uses: shivammathur/setup-php@v2
         with:
           php-version: ${{ matrix.php }}
-          extensions: dom, curl, libxml, mbstring, zip, pcntl, pdo, sqlite, pdo_sqlite, bcmath, soap, intl, gd, exif, iconv, imagick
+          extensions: dom, curl, libxml, mbstring, zip, pcntl, pdo, sqlite, pdo_sqlite, bcmath, soap, intl, gd, exif, iconv, imagick, fileinfo
           coverage: none
 
       - name: Set PHP 7.4 Mockery
-        run: composer require "mockery/mockery:>=1.2.3" --no-interaction --no-update
+        run: composer require "mockery/mockery >=1.2.3" --no-interaction --no-update
         if: matrix.php >= 7.4 && matrix.php <8.0
 
       - name: Set PHP 8 Mockery
-        run: composer require "mockery/mockery:>=1.3.3" --no-interaction --no-update
+        run: composer require "mockery/mockery >=1.3.3" --no-interaction --no-update
         if: matrix.php >= 8.0
 
       - name: Set PHP 8.1 Testbench
-        run: composer require "orchestra/testbench:^6.22.0" --no-interaction --no-update
+        run: composer require "orchestra/testbench ^6.22.0" --no-interaction --no-update
```

Comment:
Composer complained about this syntax on Windows. Using a space this way works the same.